game.py

```diff
@@ -20,239 +20,276 @@
 # Contact information:
 # Utkarsh Tiwari    iamutkarshtiwari@gmail.com
 
+import os
+from gettext import gettext as _
+
 import gi
 gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk
 import pygame
-import sys
-from gettext import gettext as _
 
-from elements import *
-from welcomescreen import *
-from scorescreen import *
+from elements import Element
+from welcomescreen import Welcomescreen
+from scorescreen import Scorescreen
+
+# Enable use of activity with `python3 game.py`
+os.environ['SUGAR_BUNDLE_ID'] = 'org.sugarlabs.2cars'
```

Comment:
Please remove this.

Reply:
Fixed

Reply:
Bundle ID isn't supposed to be set within an activity, the code for getting [bundle id](https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/activity/activity.py#L674-L679) tells you where it's supposed to be set.